paasta_tools/check_marathon_services_replication.py

```diff
@@ -198,12 +193,13 @@ def send_event_if_under_replication(
     service,
     instance,
     cluster,
-    crit_threshold,
     expected_count,
     num_available,
     soa_dir,
 ):
     full_name = compose_job_id(service, instance)
+    job_config = marathon_tools.load_marathon_service_config(service, instance, cluster)
```

Comment:
We're already looking up job_config here. Moving it so there isn't a lookup every time seems like a good idea but this will involve making monitoring_blacklist a parameter for most related functions also.